components/Leaderboard/MintNft/utils.jsx

```diff
@@ -10,7 +10,7 @@ export const getAutonolasTokenUri = (tokenUri) => (tokenUri || '').replace(patte
 
 export const mintNft = (account) => new Promise((resolve, reject) => {
   const contract = getMintContract();
-
+  if (!contract) return;
```

Comment:
i guess this should use `reject` here?

Reply:
yes ideally, it's just a temporary stub here so it doesn't annoy with errors, minting nft is currently commented out